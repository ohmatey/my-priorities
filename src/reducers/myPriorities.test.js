import myPrioritiesReducer, { myPrioritiesInitState } from './myPriorities'

describe('PrioritiesList component', () => {
  it('matches init snapshot', () => {
    const reducer = myPrioritiesReducer(myPrioritiesInitState, { type: 'nothing' })

    expect(reducer).toMatchSnapshot()
  })
})
