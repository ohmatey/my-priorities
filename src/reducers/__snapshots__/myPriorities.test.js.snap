// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrioritiesList component matches init snapshot 1`] = `Array []`;
